lib/auto-languageclient.js

```diff
@@ -251,6 +261,17 @@ export default class AutoLanguageClient {
     };
     this.postInitialization(newServer);
     connection.initialized();
+
+    const configurationKey = this.getRootConfigurationKey();
+    if (configurationKey) {
+      this._disposable.add(
+        atom.config.observe(configurationKey, config => {
+          connection.didChangeConfiguration({
+            settings: this.filterConfigurationObject(config || {}),
```

Comment:
Might be worth moving the filter/map outside the connection.onDidChangeConfiguration call and test it for null first so that people can entirely filter out sending stuff if it turns out there's nothing they want to send.

Reply:
Good point!